docs/system-admin-guide/files/external-file-storages/health-status/README.md

```diff
@@ -16,12 +16,27 @@ storages*.  In addition, administrator can manually trigger a connection validat
 
 ### Connection validation for OneDrive/SharePoint 
 
-Every file storage for OneDrive/SharePoint has the ability to run a connection test. This test is triggered manually by
-clicking on **Recheck connection** in the sidebar on the right side of the file storage's details view. This check is
-available after the file storage is fully configured.
+Every file storage for OneDrive/SharePoint has the ability to run connection checks. This test is triggered manually by clicking on **Run checks now** in the sidebar on the right side of the file storage's details view. This check is available after the file storage is fully configured.
```

Comment:
> This check is available after the file storage is fully configured.

@mereghost @Kharonus I remember that one of the tests is literally testing, whether the storage is fully configured. Do we really only _offer_ running tests after the storage is completely configured?

Reply:
I just tested this locally. I can also run checks for a not completely configured storage. I guess this sentence is here to say that this button is only available after going through the initial create dialog (which can be aborted prematurely easily).

My suggestion:

```suggestion
Every file storage for OneDrive/SharePoint has the ability to run connection checks. This test is triggered manually by clicking on **Run checks now** in the sidebar on the right side of the file storage's details view. This check is available from the UI to edit the storage.
```

(this change would have to be repeated for the nextcloud section as well)